actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -39,13 +38,27 @@ def save_image
           end
 
           def display_image
-            if ENV["CAPYBARA_INLINE_SCREENSHOT"] == "artifact"
-              "\e]1338;url=artifact://#{image_path}\a"
-            else
+            message = "[Screenshot]: #{image_path}\n"
+
+            # Environment variables have priority
+            output_type = ENV["RAILS_SYSTEM_TESTING_SCREENSHOTS_OUTPUT"] || ENV["CAPYBARA_INLINE_SCREENSHOT"]
+
+            # If running in a CI environment, default to simple
+            output_type ||= "simple" if ENV["CI"]
+
+            # Default
+            output_type ||= "inline"
+
+            case output_type
+            when "artifact"
+              message << "\e]1338;url=artifact://#{image_path}\a\n"
+            when "inline"
               name = inline_base64(File.basename(image_path))
               image = inline_base64(File.read(image_path))
-              "\e]1337;File=name=#{name};height=400px;inline=1:#{image}\a"
+              message << "\e]1337;File=name=#{name};height=400px;inline=1:#{image}\a\n"
             end
+
+            message
```

Comment:
I can't test this locally right now so maybe I'm reading it wrong - but it seems like this no longer outputs the path to the image on failure? I use the tmp path a lot when developing locally to open and look at the screenshot.